ui/site/src/site.ts

```diff
@@ -115,6 +116,8 @@ lichess.load.then(() => {
       return false;
     });
 
+    $('body').on('focusin', trapFocus);
```

Comment:
when is this necessary?

Reply:
This is necessary whenever a page could have a modal, ie whenever common/modal gets imported or when there's a special modal (ui/tournament/src/view/battle.ts and ui/analyse/src/modal.ts).

At first, I had this initialization code in common/modal. Then I realized while you need it again in ui/tournament, you _don't_ need to repeat it in ui/analyse. That's because the analyse page imports common/modal in a bunch of places. Skipping an initialization because of imports in a some faraway files felt a bit spooky to me, and I felt safer putting the code in site.ts.